packages/react-server-dom-webpack/src/__tests__/ReactFlightDOMBrowser-test.js

```diff
@@ -247,6 +247,55 @@ describe('ReactFlightDOMBrowser', () => {
     expect(container.innerHTML).toBe('<span>Hello, World!</span>');
   });
 
+  it('should resolve deduped objects within the same model root when it is blocked', async () => {
```

Comment:
maybe
```suggestion
  it.failing('should resolve deduped objects within the same model root when it is blocked', async () => {
```
so that it passes CI?

Reply:
I tried that already in #29129, but the Jest setup in the React repo does not seem to support this.